index.bs

```diff
@@ -780,9 +774,7 @@ Each {{XRReferenceSpace}} has a <dfn for="XRReferenceSpace">type</dfn>, which is
 
 An {{XRReferenceSpace}} is most frequently obtained by calling {{XRSession/requestReferenceSpace()}}, which creates an instance of an {{XRReferenceSpace}} or an interface extending it, determined by the {{XRReferenceSpaceType}} enum value passed into the call. The type indicates the tracking behavior that the reference space will exhibit:
 
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a tracking space with a [=native origin=] that tracks the position and orientation of the [=viewer=]. For {{XRSession}}s without tracking data, the [=native origin=] stays constant. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space.
-
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">position-disabled</dfn> creates an {{XRReferenceSpace}} instance.  It represents a tracking space with a [=native origin=] that tracks the position of the [=viewer=], with a constant orientation initialized based on the conventions of the underlying [=/XR device=]. It can be used to create experiences where the movement of the user is disregarded, as it will not report changes in position when used with {{XRFrame/getViewerPose()}}.
+  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">viewer</dfn> creates an {{XRReferenceSpace}} instance representing the {{XRSession}}'s [=XRSession/viewer space=].
```

Comment:
Yeah I figured it would go the other way too. Perhaps talk about it explicitly as having an native origin matching that of the [=viewer=], and then define viewerSpace as returning a {{XRReferenceSpaceType/viewer}} reference space with zero offset.